internal/cli/api/strings.go

```diff
@@ -19,7 +19,7 @@ import (
 	"strings"
 )
 
-func splitFirstSentenceOfDescription(description string) (string, string) {
+func splitShortAndLongDescription(description string) (string, string) {
 	// Split on periods that are followed by a space or end of string
 	// This approach allows us to not accidentally split verion numbers like 8.0
 	split := regexp.MustCompile(`\.(?:\s+|$)`).Split(description, -1)
```

Comment:
👀 regex compilation is usually expensive and you usually only want to do it once

Reply:
Thanks for the suggestion, will include it in a future update